### PR TITLE
CTP-2346 code review

### PR DESCRIPTION
**Pull request by aydevworks**
Overall was good to me. Just some suggestions.
May be add a README.md file to describe what this plugin does.

### CODE REVIEW

File: block_my_feedback.php [hunk 1]
```diff
@@ -145,16 +146,10 @@ public function fetch_feedback() : array {
...
                 // Hide grader, so use course image.
                 // Course image.
-                $course = new \core_course_list_element($course);
-                foreach ($course->get_course_overviewfiles() as $file) {
-                    $feedback->tutoricon = file_encode_url("$CFG->wwwroot/pluginfile.php",
-                    '/' . $file->get_contextid() .
-                    '/' . $file->get_component() .
-                    '/' . $file->get_filearea() . $file->get_filepath() . $file->get_filename());
-                }
+                $feedback->tutoricon = course_summary_exporter::get_course_image($course);
```
Comment (aydevworks) [inline block_my_feedback.php]: use core function to get course image and `file_encode_url` is deprecated.

File: block_my_feedback.php [hunk 2]
```diff
@@ -145,16 +146,10 @@ public function fetch_feedback() : array {
...
             } else {
                 // Marker details.
-                $user = $DB->get_record('user', array('id' => $f->grader));
+                $user = core_user::get_user($f->grader);
```
Comment (aydevworks) [inline block_my_feedback.php:152]: use core function.

File: version.php [hunk 3]
```diff
@@ -25,6 +25,8 @@
 defined('MOODLE_INTERNAL') || die();
 
 $plugin->version   = 2023031400;        // The current plugin version (Date: YYYYMMDDXX).
+$plugin->release   = '1.0';
```
Comment (aydevworks) [inline version.php:28]: Looks like more completed version.php
https://moodledev.io/docs/apis/commonfiles/version.php

**Review by stuartlamour — APPROVED** [review]
Smoke tested changes, thanks @aydevworks

**Pull request merged**